props/prop_base.h

```diff
@@ -545,17 +545,17 @@ class PropBase : CommandParser, protected Looper, protected SaberBase, public Mo
     float ret = blade_id.id();
 
     if (announce) {
-      PVLOG_STATUS << "BLADE ID: " << ret << "\n";
+      STDOUT << "BLADE ID: " << ret << "\n";
 #ifdef SPEAK_BLADE_ID
     SpeakBladeID(ret);
 #endif // SPEAK_BLADE_ID
     }
 #ifdef BLADE_DETECT_PIN
     if (!blade_detected_) {
-      PVLOG_STATUS << "NO ";
+      STDOUT << "NO ";
       ret += NO_BLADE;
     }
-    PVLOG_STATUS << "Blade Detected\n";
```

Comment:
Isn't it enough to just change line 548? (And add a comment that specifies why it needs to be STDOUT.)

Reply:
Sure, done.